### ISSUE TITLE
How to use with Expo?

### ISSUE THREAD

**Issue by taitsmp** [jasancheg/react-native-prepare-svg#1]
Thanks for releasing react-navitve-prepare-svg.  I looked into how to use this without `Svgx`.  Am I right in assuming that if I want to use the json file produced by `rn-prepare-svg` with react-native-svg I to either to manually rewrite the SVG-related tags by hand or rely on the `Svgx` class in react-native-svgx?  There's no easy way use the json file directly with the `Svg` component from react-native-svg. 

Let me know if I missed something.

**Comment by jasancheg** [comment]
Hello @taitsmp, thank you very much!

* Please excuse delay in response, I was absent for a long time, my son was about to be born.

In order to use it with SVG you must create a component that allows you to filter the data of the JSON file in a custom way. SVGX is a wrapper for the SVG component that does this.

The react-navitve-prepare-svg and react-native-svgx packages are closely related, both were created with the purpose of solving an immediate task of a personal work. However I can see that there are people interested in this functionality so I will think of a more flexible version for the community very soon.

**Comment by taitsmp** [comment]
Makes sense.  Excited to play with `Svgx` more.  Thanks for building and releasing this.
